### ISSUE TITLE
Any1 Feel like doing a traceback problem?

### ISSUE THREAD

**Issue by ShiftingKill** [InstaPy/InstaPy#1895]
So after reading through the traceback it looks like selenium webdriver says:
session deleted because of page crash
from unknown error: cannot determine loading status
from tab crashed

Is this just a selenium update needed?

Session info: chrome v 65.0..181
Driver info: chromedriver 2.37...315

**Comment by MichalGallovic** [comment]
I'm experiencing the same problem with this error message:
```
unfollow loop error Message: unknown error: session deleted because of page crash
from tab crashed
  (Session info: chrome=65.0.3325.181)
  (Driver info: chromedriver=2.37.544337 (8c0344a12e552148c185f7d5117db1f28d6c9e85),platform=Mac OS X 10.9.5 x86_64)

Could not locate Chrome: Message: no such session
  (Driver info: chromedriver=2.37.544337 (8c0344a12e552148c185f7d5117db1f28d6c9e85),platform=Mac OS X 10.9.5 x86_64)
```

running with python3.6

**Comment by MichalGallovic** [comment]
Interesting is that I have no problem with unfollowing on different profile, using the same machine with the same env.

I tried doing fresh install, selenium upgrade, chromedriver upgrade. Did not help.

**Comment by uluQulu** [comment]
Which features are you using and with which parameters?

**Comment by sionking** [comment]
I get it on all of my users, I just restart the application. it do happen once in 2-3 days.

**Comment by sionking** [comment]
Using Firefox is much more stable

**Comment by amauryleproux** [comment]
@ShiftingKill did you find the issue?

**Comment by ShiftingKill** [comment]
@amauryleproux No I did not, but it was fixed. I can't remember as it has been some time, but I think I just had to really make sure my python environment variables were good, git and github fully updated, and not just mashing stuff together. I think I finally got InstaPy down to a science. Before I was probably downloading and running things I shouldn't have done, such as doing the manual install AND the automatic install (windows.ps1 I think), or mashing parts of them together. Anyways, the years of playing with LEGOs helped, I put InstaPy together correctly and it hasn't failed since. Computers are strange beasts...
